Program.cs

```diff
@@ -177,7 +184,7 @@ class P2pEndPoint
             internal IPEndPoint Internal;
         }
 
-        static P2pEndPoint GetExternalEndPoint(Socket socket)
+        static P2pEndPoint GetExternalEndPoint(Socket socket, bool printMessages = true)

```

Comment:
this change above seems unrelated to the task at hand

Reply:
Ok, I rollback this change.